packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -516,7 +516,10 @@ class _DropdownRoute<T> extends PopupRoute<_DropdownRouteResult<T>> {
   // selected item is aligned with the button's vertical center, as far as
   // that's possible given availableHeight.
   _MenuLimits getMenuLimits(Rect buttonRect, double availableHeight, int index) {
-    final double maxMenuHeight = availableHeight - 2.0 * _kMenuItemHeight;
+    double maxMenuHeight = availableHeight - 2.0 * _kMenuItemHeight;
```

Comment:
`maxMenuHeight` v `menuMaxHeight` is a bit confusing.
What if we made this one `computedMaxHeight`? Or something like that?